### PR TITLE
🔒️ Fix token security.

### PR DESCRIPTION
**Pull request by nxtdv**
Added a README.md to know how to install it.

### CODE REVIEW

**Review by martin-fullstacks — COMMENTED** [review]
Why are you deleting the settings of the client ?

**Comment by nxtdv** [comment]
> Why are you deleting the settings of the client ?

https://www.npmjs.com/package/dotenv

File: index.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+  messageCacheMaxSize: 10,
+  restTimeOffset: 0,
+  restWsBridgetimeout: 100,
+  shards: "auto",
+  allowedMentions: {
+    parse: ["roles", "users", "everyone"],
+    repliedUser: true,
+  },
+  partials: ["MESSAGE", "CHANNEL", "REACTION"],
+  intents: 32767,
```
Comment (a9ex) [inline index.js:16]: By the way, it would be better to put the intents in an Array for a better understanding

Comment (a9ex) [reply index.js:16]: Let's delete some Intents that are useless now and in the future
```suggestion
  intents: [
    'GUILDS',
    'GUILD_MEMBERS',
    'GUILD_BANS',
    'GUILD_EMOJIS_AND_STICKERS',
    'GUILD_VOICE_STATES',
    'GUILD_MESSAGES',
    'GUILD_MESSAGE_REACTIONS',
    'DIRECT_MESSAGES',
  ],
```

**Review by a9ex — COMMENTED** [review]
It looks good to me, we could just correct a few things :

**Pull request merged**